webapp/index.php

```diff
@@ -1,280 +1,216 @@
+        <div class="datum" id="study_month">
+          <p class="dataTitle">Month</p>
+          <p class="data_hour">
+            <?php
+            if ($hours_month["month"] === null) {
+              echo 0;
+            } else {
+              echo $hours_month["month"];
+            }
+            ?>
+          </p>
+          <p class="hour">hour</p>
+        </div>
+        <div class="datum" id="study_total">
+          <p class="dataTitle">Total</p>
+          <p class="data_hour">
+
+            <?php
+            if ($hours_total["total"] === null) {
+              echo 0;
+            } else {
+              echo $hours_total["total"];
+            }
+            ?>
+
+          </p>
+          <p class="hour">hour</p>
         </div>
       </div>
-      <div id="pieChartArea">
-        <div class="chartContainer" id="container_pieChart_language">
-          <p class="pieChartTitle">学習言語</p>
-          <div id="pieChart_language"></div>
-          <div class="pieChartElementsArea">
-            <?php foreach($hours_language as $language): ?>
+      <div class="chartContainer">
+        <div id="columnChart"></div>
+      </div>
+    </div>
+    <div id="pieChartArea">
+      <div class="chartContainer" id="container_pieChart_language">
+        <p class="pieChartTitle">学習言語</p>
+        <div id="pieChart_language"></div>
+        <div class="pieChartElementsArea">
+
+          <?php foreach ($hours_language as $language) : ?>
             <div class="pieChartElements">
               <?= $language["language"], "　", $language["hours"] ?>
             </div>
-            <?php endforeach ?>
-          </div>
+          <?php endforeach ?>
+
         </div>
-        <div class="chartContainer" id="container_pieChart_contents">
-          <p class="pieChartTitle">学習コンテンツ</p>
-          <div id="pieChart_contents"></div>
-          <div class="pieChartElementsArea">
-            <?php foreach($hours_content as $content): ?>
+      </div>
+      <div class="chartContainer" id="container_pieChart_contents">
+        <p class="pieChartTitle">学習コンテンツ</p>
+        <div id="pieChart_contents"></div>
+        <div class="pieChartElementsArea">
+          <?php foreach ($hours_content as $content) : ?>
             <div class="pieChartElements">
               <?= $content["content"], "　", $content["hours"] ?>
             </div>
-            <?php endforeach ?>
-          </div>
+          <?php endforeach ?>
         </div>
       </div>
     </div>
-    <div class="month">
-      <section class="prev"></section>
-      <p><?= $year ?>年 <?= $month ?>月</p>
-      <section class="next"></section>
-    </div>
+  </div>
+  <div class="month">
+    <section class="prev"></section>
+    <p><?= date("Y") ?>年 <?= date("m") ?>月</p>
+    <section class="next"></section>
+  </div>
 
-    <button onclick="showModal()" id = "button_sp" class="button">記録・投稿</button>
+  <button onclick="showModal()" id="button_sp" class="button">記録・投稿</button>
```

Comment:
onclick属性は便利ですが、一つしか処理を登録できないのでaddEvent~~~を利用してください
```suggestion
  <button  id="button_sp" class="button">記録・投稿</button>
```

Reply:
一応治したけど、jsきったない